ci-operator/config/openshift/hac-core/openshift-hac-core-main.yaml

```diff
@@ -16,6 +16,37 @@ tests:
   commands: ./test.sh
   container:
     from: src
+- as: release-build-repo
+  commands: |
+    mkdir -p /etc/ssh
+    ssh-keyscan github.com >> /etc/ssh/ssh_known_hosts
+    chmod 600 /tmp/secret/SSH_PRIVATE_KEY
+    echo "Host github.com
+      HostName github.com
+      User git
+      AddKeysToAgent yes
+      IdentityFile /tmp/secret/SSH_PRIVATE_KEY" >> /etc/ssh/ssh_config
+    ./build.sh
+    ./release-build-repo.sh
+  container:
+    from: src
+  postsubmit: true
+  secret:
+    namespace: test-credentials
```

Comment:
@karelhala  I think `ci/prow/ci-operator-registry` doesn't like this or your other use of `namespace` 🤔 

[Logs Link](https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/24323/pull-ci-openshift-release-master-ci-operator-registry/1468578265902878720)

Error unravelled. 

```
{
  "component":"ci-operator-configresolver",
  "file":"/go/src/github.com/openshift/ci-tools/cmd/ci-operator-configresolver/main.go:130",
  "func":"main.main",
  "level":"fatal",
  "msg":"Failed to get config agent: failed to laod config: loading config failed: failed to load ci-operator config (invalid configuration in file ci-operator/config/openshift/hac-core/openshift-hac-core-main.yaml: error unmarshaling JSON: while decoding JSON: json: unknown field \"namespace\"
    value:
    binary_build_commands: ./build.sh
    build_root:
      image_stream_tag:
        name: release
        namespace: openshift
        tag: tectonic-console-builder-v23
    resources:
      '*':
        limits:
          memory: 4Gi
        requests:
          cpu: 100m
          memory: 200Mi
    tests:
    - as: test
      commands: ./test.sh
      container:
        from: src
    - as: release-build-repo
      commands: |
        mkdir -p /etc/ssh
        ssh-keyscan github.com \u003e\u003e /etc/ssh/ssh_known_hosts
        chmod 600 /tmp/secret/SSH_PRIVATE_KEY
        echo \"Host github.com
          HostName github.com
          User git
          AddKeysToAgent yes
          IdentityFile /tmp/secret/SSH_PRIVATE_KEY\" \u003e\u003e /etc/ssh/ssh_config
        ./build.sh
        ./release-build-repo.sh
      container:
        from: src
      postsubmit: true
      secret:
        namespace: test-credentials
        name: hac-core-deploy-tokens
        mount_path: /tmp/secret
    - as: release-image
      commands: |
        export DOCKER_TOKEN=$(cat /tmp/secret/DOCKER_TOKEN)
        ./build.sh
        ./release-image.sh
      container:
        from: src
      postsubmit: true
      secret:
        namespace: test-credentials
        name: hac-core-deploy-tokens
        mount_path: /tmp/secret
    zz_generated_metadata:
      branch: main
      org: openshift
      repo: hac-core
    )",
  "severity":"fatal",
  "time":"2021-12-08T13:49:28Z"
}
```

Reply:
Aaaah yeah! I've used secret instead of credentials